### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sstadick**
This is an awesome list, I learned about a few new tools in here. Thanks for adding `hck`!

### CODE REVIEW

**Review by matu3ba — CHANGES_REQUESTED** [review]
can you change the other typo as well? `delimmiter` is not correct. sorry for the inconvenience.

File: README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ and [@ghuls](https://github.com/zaiste/zaiste.net/issues/7) for broot, git-inter
 | exa       | exa       | `ls`                                                                | [exa](https://github.com/ogham/exa)                   |  exa    |
 | fd        | fd-find   | `find`                                                              | [fd](https://github.com/sharkdp/fd)                   |  fd     |
 | fd        | fd-find   | `find`                                                              | [fd](https://github.com/sharkdp/fd)                   |  fd     |
-| hck       | hck       | `cat` with decompression, column reordering and regex as delimmiter | [hck](https://github.com/sstadick/hck)                |  hck    |
+| hck       | hck       | `cut` with decompression, column reordering and regex as delimmiter | [hck](https://github.com/sstadick/hck)                |  hck    |
```
Comment (sstadick) [inline README.md]:
```suggestion
| hck       | hck       | `cut` with decompression, column reordering and regex as delimiter | [hck](https://github.com/sstadick/hck)                |  hck    |
```

**Comment by sstadick** [comment]
Fixed!

**Comment by matu3ba** [comment]
Thanks for the fix @sstadick.

**Pull request merged**